python/tests/integration/arcticdb/test_arctic.py

```diff
@@ -1050,12 +1050,27 @@ def test_get_uri(object_storage_uri_incl_bucket):
 
 @pytest.mark.skipif(not AZURE_SUPPORT, reason="Pending Azure Storge Conda support")
 def test_azure_no_ca_path(azurite_azure_test_connection_setting):
-    (endpoint, container, credential_name, credential_key, ca_cert_path) = azurite_azure_test_connection_setting
-    ac = Arctic(
+    endpoint, container, credential_name, credential_key, _ = azurite_azure_test_connection_setting
+    Arctic(
         f"azure://DefaultEndpointsProtocol=http;AccountName={credential_name};AccountKey={credential_key};BlobEndpoint={endpoint}/{credential_name};Container={container}"
     )
 
 
+def test_azure_sas_token(azure_account_sas_token, azurite_azure_test_connection_setting):
+    endpoint, container, credential_name, _, _ = azurite_azure_test_connection_setting
+    ac = Arctic(
+        f"azure://DefaultEndpointsProtocol=http;SharedAccessSignature={azure_account_sas_token};BlobEndpoint={endpoint}/{credential_name};Container={container}"
```

Comment:
We should document how to do this in the website

Reply:
Yeah exactly, a little example in the docs showing how to connect to Azure with SAS

Reply:
Maybe the best is by adding the link of below official guidelines for user reference?
Create token:
https://learn.microsoft.com/en-us/azure/storage/common/storage-account-sas-create-python
Connection string:
https://learn.microsoft.com/en-us/azure/storage/common/storage-configure-connection-string#create-a-connection-string-using-a-shared-access-signature